src/rest.ts

```diff
@@ -33,6 +33,22 @@ export type RequestOptions = {
   onResponse?: (r: ApiResponse) => void;
 };
 
+export type PollOptions = {
+  overheadRate?: number;
+  startTime?: number;
```

Comment:
`startTime` looks kinda weird, doesn't it?

Reply:
So do we want to hardcode it in the `pollWithOverhead` function to `Date.now`, or we should just not expose it in the `exec` and `pollTransaction` options?

Reply:
Hardcode in `pollWithOverhead`